Package.swift

```diff
@@ -1,6 +1,22 @@
 // swift-tools-version:5.9
 
 import PackageDescription
+import Foundation
+import WinSDK
+
+let swiftc = "swiftc.exe".withCString(encodedAs: UTF16.self) { pwszCompiler in
+  let dwLength = SearchPathW(nil, pwszCompiler, nil, 0, nil, nil)
+  return withUnsafeTemporaryAllocation(of: WCHAR.self, capacity: Int(dwLength) + 1) {
+    _ = SearchPathW(nil, pwszCompiler, nil, dwLength + 1, $0.baseAddress, nil)
+    return String(decodingCString: $0.baseAddress!, as: UTF16.self)
+  }
+}
+let include = URL(fileURLWithPath: swiftc).deletingLastPathComponent()
+                                          .deletingLastPathComponent()
+                                          .appendingPathComponent("include")
+                                          .withUnsafeFileSystemRepresentation {
```

Comment:
I think there can be some edge cases with trying to convert a Windows file path to UTF-8, but probably not something we will run into.